packages/vrealize-developer-tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
```

Comment:
Lets keep this file in the root of the repo. It is not specific to the vscode extension, but the whole repository (also GitHub provides links at several places that lead to the root CONTRIBUTING.md file)